providers/azure/azure.go

```diff
@@ -318,15 +293,8 @@ func (b *Bucket) Exists(ctx context.Context, name string) (bool, error) {
 // Upload the contents of the reader as an object into the bucket.
 func (b *Bucket) Upload(ctx context.Context, name string, r io.Reader) error {
 	level.Debug(b.logger).Log("msg", "uploading blob", "blob", name)
-	blobClient, err := b.containerClient.NewBlockBlobClient(name)
-	if err != nil {
-		return err
-	}
-	opt := azblob.UploadStreamOptions{
-		BufferSize: 3 * 1024 * 1024,
-		MaxBuffers: 4,
```

Comment:
Need to figure out what these are called now. Additionally the values should be documented to specify why they are being set in the first place.

Reply:
The values have been renamed BlockSize and Concurrency. I think that in a future PR we should consider making these configurable or explaining why this is a good value.